packages/tic-tac-toe/app/services/ttt-channel-client.ts

```diff
@@ -1,10 +1,10 @@
 import Service from '@ember/service';
 import {
   ChannelResult,
-  Message,
   ChannelClientInterface,
   UnsubscribeFunction
 } from '@statechannels/channel-client';
+import {Message} from '@statechannels/client-api-schema';
```

Comment:
This appears to be a real dependency, not a dev dependency. Is there a reason you only include it as a dev dependency?

Reply:
Hmm, doesn't really matter but this is a TS type import. So if it's important to anyone I can move it to dependencies.

Reply:
@andrewgordstewart I think type information imports make sense as dev dependencies

Reply:
It depends on the distributed file in the `package.json` but in the case of `tic-tac-toe` it's a JS bundle, so seems reasonable to make type information a dev dependency

Reply:
Ah, I hadn't clued in that it it was for type information only.

Reply:
So can we merge this in? 😁